build.sh

```
@@ -120,8 +120,14 @@ if [[ "$*" == *"--frontend=typescript"* ]]; then
 	cd "$LOCA/frontend-ts/" || exit 1
 	bun install $BUNFLAGS
 	noti "Transpiling and copying to Lumina server..."
-	bun $BUNFLAGS build "$LOCA/frontend-ts/app.ts" --minify --target=browser --outdir "$LOCA/backend/priv/generated/js/" --sourcemap=linked
-	bun $BUNFLAGS "$LOCA/tobundle.ts" -- js-1 "$LOCA/backend/priv/generated/js/app.js"
+	bun $BUNFLAGS build "$LOCA/frontend-ts/app.ts" --minify --target=browser --outdir "$LOCA/backend/priv/generated/js/" --sourcemap=linked || {
```

Shellcheck (0.9.0-1)

[info] Double quote to prevent globbing and word splitting.

(SC2086)